cmds/core/init/init_linux.go

```diff
@@ -6,17 +6,57 @@ package main
 
 import (
 	"log"
+	"os"
+	"path/filepath"
 	"strconv"
+	"strings"
 	"syscall"
 
 	"github.com/u-root/u-root/pkg/cmdline"
+	"github.com/u-root/u-root/pkg/kmodule"
 )
 
+// installModules installs kernel modules (.ko files) from /lib/modules.
+// Useful for modules that need to be loaded for boot (ie a network
+// driver needed for netboot)
+func installModules() {
+	modulePattern := "/lib/modules/*.ko"
+	files, err := filepath.Glob(modulePattern)
+	if err != nil {
+		log.Printf("installModules: error finding files at %q: %v", modulePattern, err)
+		return
+	}
+	if len(files) == 0 {
+		log.Printf("installModules: no modules matching pattern %q. Not installing any modules", modulePattern)
```

Comment:
I don't think we need any more noise on boot. Maybe we can not do this print, since the common case is there won't be modules. We're finding that even small error messages are confusing to people.

Reply:
Fixed in latest.